spec/support/resolv_helpers.rb

```diff
@@ -1,6 +1,6 @@
 module ResolvHelpers
   def stub_resolv(name, address)
-    dns = double
+    dns = double("dns")
```

Comment:
oo sure. happy to update. This change was helpful for confirming that this was the double that was leaking, btw (just mentioning in case it wasn't clear why this change was included)